### PR TITLE
fix: TOS-652 When clicked on "Cancel" button in the element creation page of Mobile inspector it doesn't navigate back to Elements List page instead displays blank space

### PR DESCRIPTION
**Pull request by manikanta-reddy-t**

### CODE REVIEW

File: ui/src/app/components/webcomponents/element-form.component.ts [hunk 1]
```diff
@@ -329,6 +329,21 @@ export class ElementFormComponent extends BaseComponent implements OnInit {
...
+      version.id=this.versionId;
+      let sendDetails = {
+        versionId: this.versionId,
+        version:version,
+        isNew: true,
+        testCaseId: this.testCaseId,
+        testCaseResultId: this.options.testCaseResultId,
+        isStepRecordView: this.options.isStepRecordView
+      }
+      this.mobileRecorderEventService.suggestionContent.next(Object.assign(sendDetails, {
```
Comment (guganabu) [inline ui/src/app/components/webcomponents/element-form.component.ts:343]: you can use spread operator to export an object {...sendDetails, content: <>} @manikanta-reddy-t

Comment (manikanta-reddy-t) [reply ui/src/app/components/webcomponents/element-form.component.ts]: updated according to comments

File: ui/src/app/components/webcomponents/element-form.component.ts [hunk 2]
```diff
@@ -329,6 +329,20 @@ export class ElementFormComponent extends BaseComponent implements OnInit {
     this.chromeRecorderService.elementCallBack = undefined;
     this.chromeRecorderService.stopSpying();
     this.formDetails ? this.mobileRecorderEventService.setEmptyAction() : ( isClose? this.dialogRef.close(this.reviewSubmittedElement): null );
+    if(isClose && this.options.isStepRecordView){
+      let version=new WorkspaceVersion();
+      version.id=this.versionId;
+      let sendDetails = {
+        versionId: this.versionId,
+        version:version,
```
Comment (guganabu) [inline ui/src/app/components/webcomponents/element-form.component.ts:337]: does it require, to have version and versionId in two different prop? @manikanta-reddy-t

Comment (manikanta-reddy-t) [reply ui/src/app/components/webcomponents/element-form.component.ts]: updated

**Review by guganabu — APPROVED** [review]

**Pull request merged**